mms/redis_conf.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+This module parses a Redis configuration file to ensure the same parameters
+used to initialize Redis are used by the Redis object to access Redis.
+"""
+
+CONFIG_MAP = {"bind": "host", "port": "port", "requirepass": "password"}
+
+
+class RedisConfParser(object):
```

Comment:
Where is this being used?

Reply:
part of another pr, can move it to this one if necessary